debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (22.4.2-0ubuntu2) lunar; urgency=medium
+
+  * Upstream snapshot based on upstream/main at 040090a3.
+    - Bugs fixed in this snapshot: (LP: #1994980, #1998655, #1999042)
```

Comment:
Is this line wrapping expected? I would expect something like:

```
    - Bugs fixed in this snapshot: (LP: #1994980, #1998655, #1999042,
    #1997559)
```

Reply:
Yes this 'formatting' is intentional because the changes file which is generated by dpkg-buildpackage has a limited regex for matching `Launchpad-Bugs-Fixed:` that cannot parse multi-line LP: #123,\n#456. So a build with this mult-line suggestion would redact 1997559 from the `Launchpad-Bugs-Fixed:` in the sources file.

Reply:
With this existing d/changelog we get the following in ../out/cloud-init_22.4.2-0ubuntu2_source.changes:
`Launchpad-Bugs-Fixed: 1994980 1997559 1998655 1999042`

with your amended suggestion:
`Launchpad-Bugs-Fixed: 1994980 1998655 1999042`